src/components/ProductCategories/ProductCategories.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import AliceCarousel from "react-alice-carousel";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const Img = styled.img`
+  width: 100%;
+`;
+
+const Title = styled.h1`
+  text-align: center;
+`;
+
+ProductCategories.propTypes = {
+  categories: PropTypes.array,
+};
+
+ProductCategories.defaultProps = {
+  categories: [],
+};
+
+function CategoryItem({ category }) {
+  const {
+    data: {
+      name,
+      main_image: { url, alt },
+    },
+  } = category;
+
+  return (
+    <div>
+      <h4 style={{ textAlign: "center" }}>{name}</h4>
+      <Img alt={alt} src={url} />
+    </div>
+  );
+}
+
+function ProductCategories({ categories }) {
+  const items = categories.map((category, index) => (
+    <CategoryItem key={index} category={category} data-value={index} />
+  ));
+
+  return (
+    <div className="product-categories">
+      <Title>Product Categories</Title>
+      <AliceCarousel disableButtonsControls items={items} />
+    </div>
```

Comment:
I feel this code could be converted into a reusable component. If you see in [this previous code](https://github.com/gustavo807/wa-react-2021-q4-capstone-project/pull/1/files#diff-0e1716f2cdaf3a608717b5028bb45267123fd5365f83ebbe28f14c6536694e33R46) looks very similar to this one. The only difference I notice is the `categories` prop vs `banners` along with other namings.